docs/readme.md

```diff
@@ -4,17 +4,20 @@
 
 [![API status](https://badgen.net/uptime-robot/status/m793274556-25c5e9bbab0297d91cda7134)](https://stats.uptimerobot.com/57wNLs39M/793274556)
 
-Because it wraps [an API](https://github.com/public-transport/hafas-client/blob/6/readme.md#background) of [Deutsche Bahn](https://de.wikipedia.org/wiki/Deutsche_Bahn), it **includes most of the long-distance and regional traffic, as well as some international trains and local buses**. Essentially, it returns whatever data the [*DB Navigator* app](https://www.bahn.de/p/view/service/mobile/db-navigator.shtml) shows, **including realtime delays and disruptions**.
+Because it wraps [an API](https://github.com/public-transport/hafas-client/blob/6/readme.md#background) of [Deutsche Bahn](https://de.wikipedia.org/wiki/Deutsche_Bahn), it **includes most of the long-distance and regional traffic, as well as some international trains and local buses**. Essentially, it returns whatever data the [*DB Navigator* app](https://www.bahn.de/p/view/service/mobile/db-navigator.shtml) shows*, **including realtime delays and disruptions**.
+
+*When comparing realtime information for past journeys, sometimes differing results can be observed between the API response and what the DB Navigator app shows. This might be due to implementation details in the app.*
```

Comment:
This projects uses "mgate" HAFAS, this was used in the "old" DB Navigator, now known as Business DB Navigator.
The current DB Navigator uses a different HAFAS backend (also known as Vendo) and it's own abstraction on HAFAS. (only used by DB, mgate is a white label solution by HAFAS)
The configurations for the current DB Navigator HAFAS and the business DB Navigator one thats used by this project differs. Therefore it may show different times.

Reply:
I see, I thought the backend stayed the same. This is in contradiction to what @derhuerst said in [his comment](https://github.com/derhuerst/db-rest/issues/42#issuecomment-1970044955), no? If that's the case, and the access to the RSI::Journeys API is possible, shouldn't this repo switch to using the Journeys API, so we get identical results again?
Or is Vendo != RSI:Journeys? Not that I think it's a big deal. I'm getting into details here that really don't matter for my cause and I'm obviously very grateful for all the projects provided.

Back to the task on hand, I gave it another shot.

> *When comparing results from this API to what the DB Navigator app shows there might be occasional differences due to them utilizing different, not 100% identical backends.*

Reply:
Vendo is HAFAS, but not the HAFAS used here. RIS::Journeys is something different. (but closer to the source of all information)

Reply:
*Technically* (and only as far as I know, @marudor knows much better), Vendo is Deutsche Bahn's rewrite of the their shop and mobile apps, so the mobile apps and their specific backends could be called Vendo. Apparently, in the backend behind some Vendo-specific services, they're based on a HAFAS system (which is in turn built by [HaCon](https://www.hacon.de)), however that doesn't mean that we can access any of them.

We *can* – as in it's technically possible – use Deutsche Bahn's "old" `mgate.exe`-style HAFAS API, which was used by the old DB Navigator implementation (now apparently Business Navigator).

RIS::Journeys is something else entirely, even though it covers some similar use cases. It's not intended for public consumption, but it *might* be that you get access to it (just speculating here).

Reply:
RIS::Journeys is used in several information systems DB has. https://zugportal.de/ for instance. But it's also used in RIS::Fahrzeug (so DB Regio trains and ICE trains)
ICE Portal is another information system that uses it.
And just like @derhuerst says, Vendo is usually referring the bahn.de/DB Navigator rewrite that happened the last years and went live last year.
They still mainly use HAFAS as backend. (If its hosted by DB, customized by DB or used as a Service by HaCon does not really matter here)
But `mgate.exe` and current `bahn.de/DB Navigator` use slightly different HAFAS.